# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/angle/checkout/src/compiler/translator/AtomicCounterFunctionHLSL.h
# Commit: 3a91885d47e0
# Full Hash: 3a91885d47e0ef7fba9749562d52300eea67c651
# Author: Miko Mynttinen <mikokm@gmail.com>
# Date: 2019-11-11 21:52:52
# Regressor Bug: 1578576
# File Overlap Count: 4
# Description:
#   Bug 1578576 - Part 2: Update to ANGLE 3865 r=jgilbert
#   
#   Depends on D44561
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44579
# ==============================================================================

diff -r 77cee77c0307 -r 3a91885d47e0 gfx/angle/checkout/src/compiler/translator/AtomicCounterFunctionHLSL.h
--- a/gfx/angle/checkout/src/compiler/translator/AtomicCounterFunctionHLSL.h	Fri Nov 08 16:41:42 2019 +0000
+++ b/gfx/angle/checkout/src/compiler/translator/AtomicCounterFunctionHLSL.h	Fri Nov 08 17:06:00 2019 +0000
@@ -24,6 +24,8 @@
 class AtomicCounterFunctionHLSL final : angle::NonCopyable
 {
   public:
+    AtomicCounterFunctionHLSL(bool forceResolution);
+
     ImmutableString useAtomicCounterFunction(const ImmutableString &name);
 
     void atomicCounterFunctionHeader(TInfoSinkBase &out);
@@ -38,6 +40,7 @@
     };
 
     std::map<ImmutableString, AtomicCounterFunction> mAtomicCounterFunctions;
+    bool mForceResolution;
 };
 
 ImmutableString getAtomicCounterNameForBinding(int binding);